lib/Installation/Partitioner/Libstorage/ProposalSettingsDialog.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: TBD
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+package Installation::Partitioner::Libstorage::ProposalSettingsDialog;
+use strict;
+use warnings FATAL => 'all';
+
+use PageInteractions;
+
+use parent 'Exporter';
+use Exporter;
+
+our @EXPORT = qw(
+    select_encrypted_lvm_based_proposal_radiobutton
+    apply_proposal_settings
+);
+
+my $PROPOSAL_SETTINGS_DIALOG = 'inst-partition-radio-buttons';
+
+sub select_encrypted_lvm_based_proposal_radiobutton {
+    find($PROPOSAL_SETTINGS_DIALOG);
+    send_keys('alt-e', 'Select "Encrypted LVM-based Proposal" radiobutton.');
+}
+
+sub apply_proposal_settings {
```

Comment:
Perhaps call it press_ok or something like that to avoid confuse with "apply" buttons.